web/gds-user-ui/cypress.config.ts

```diff
@@ -0,0 +1,63 @@
+import { defineConfig } from "cypress";
+import webpack from "@cypress/webpack-preprocessor";
+import { addCucumberPreprocessorPlugin } from "@badeball/cypress-cucumber-preprocessor";
+
+async function setupNodeEvents(
+  on: Cypress.PluginEvents,
+  config: Cypress.PluginConfigOptions
+): Promise<Cypress.PluginConfigOptions> {
+  await addCucumberPreprocessorPlugin(on, config);
+
+  on(
+    "file:preprocessor",
+    webpack({
+      webpackOptions: {
+        resolve: {
+          extensions: [".ts", ".js"],
+        },
+        module: {
+          rules: [
+            {
+              test: /\.ts$/,
+              exclude: [/node_modules/],
+              use: [
+                {
+                  loader: "ts-loader",
+                },
+              ],
+            },
+            {
+              test: /\.feature$/,
+              use: [
+                {
+                  loader: "@badeball/cypress-cucumber-preprocessor/webpack",
+                  options: config,
+                },
+              ],
+            },
+          ],
+        },
+      },
+    })
+  );
+
+  return config;
+}
+
+export default defineConfig({
+  e2e: {
+    baseUrl: "http://localhost:3000",
+    specPattern: "**/*.feature",
+    setupNodeEvents,
+    env: {
+        API_URL: "http://localhost:4437/v1"
+    },
+  },
+  reporter: "mochawesome",
```

Comment:
I ran into an issue with testing generating reports. I'll create a separate story and handle that later.